src/include/security.h

```diff
@@ -199,6 +200,14 @@ pgmoneta_close_ssl(SSL* ssl);
 int
 pgmoneta_get_hash_algorithm(char* algorithm);
 
+/**
+ * Extract server parameters recevied during the latest authentication
+ * @param server_parameters The deque to store the extracted server parameters
+ * @return 0 on success, otherwise 1
+ */
+int
+pgmoneta_extract_server_parameters(struct deque* server_parameters);
```

Comment:
This function should do the `deque` so `struct deque**`

Reply:
`NULL` if `1` is returned

Reply:
I've updated it. Could you kindly review it and let me know if it aligns with your intent?